recipes/timsort/all/test_package/CMakeLists.txt

```diff
@@ -5,4 +5,11 @@ find_package(gfx-timsort REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
 target_link_libraries(${PROJECT_NAME} PRIVATE gfx::timsort)
-target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
+
+if(gfx-timsort_VERSION VERSION_LESS "2.0.0")
+  target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_98)
+elseif(gfx-timsort_VERSION VERSION_LESS "3.0.0")
+  target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_11)
+else()
+  target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_20)
```

Comment:
Since cxx_std_20 requires 3.12, would you update `cmake_minimum_required`?

https://cmake.org/cmake/help/latest/prop_gbl/CMAKE_CXX_KNOWN_FEATURES.html

Reply:
Thinking about it again maybe these lines aren't be useful: shouldn't the conan-generated targets assign the correct compile features for the standard version?

I introduced them on your suggestion to fix the previous problem, but maybe constraining the compiler versions was enough.

Reply:
For Conan 2.x is all solved, the CMakeDeps, etc. will require 3.15. Plus, cppstd is defined by default there, so passing target_compile_features should no longer needed. I guess it's okay keeping the current, otherwise it would fail to build with Conan 2.x in case running an older version of cmake

Reply:
By "the current", do you mean as it was before this MR, or the `target_compile_features` currently in the MR?

Reply:
I get errors when removing `target_compile_features` apparently, guess I incorrectly interpreted your message.

Reply:
Bump because it's been half a year and I'm still unsure what you meant.